src/services/Config.php

```diff
@@ -915,6 +915,34 @@ private function _loadConfigSettings(string $category)
             }
         }
 
```

Comment:
@takobell I originally had the logic there, but realized that `Config::CATEGORY_DB` is reference throughout the application, and not just in `config/components/common.php`. These files include:

- `base/ApplicationTrait.php`
- `db/Connection.php`
- `db/mysql/Schema.php`
- `db/pgsql/Schema.php`
- `et/EtTransport.php`
- `helpers/ChartHelper.php`
- `migrations/Install.php`

Putting the logic in `services/Config.php` pretty much centralizes the processing so that by the time the other classes need to retrieve specific variables, such as the driver, the database url is already parsed and there is not need to re-parse.

I'm still open to putting this in a better place; this just seemed like the most central place.